docs/laborok/02-android-ui-compose/index.md

```diff
@@ -295,6 +295,7 @@ fun LoginScreen() {
                 keyboardType = KeyboardType.Password
             ),
             singleLine = true,
+            visualTransformation = PasswordVisualTransformation(),
```

Comment:
😻

Reply:
@gazdilaci ez egy menő quality of life improvement, ha a kolléga egy hallgató, akkor erre adhatunk 1 pontot szerintem.

Reply:
A kolléga hallgató :sunglasses:

Reply:
majd egy neptunt írj be a descriptionbe pls